src/ecdsa/test_ecdsa.py

```diff
@@ -30,6 +30,31 @@ def test_signature_validity(Msg, Qx, Qy, R, S, expected):
       raise TestFailure("*** Signature test failed: got %s, expected %s." % \
                         (got, expected))
 
+  def test_pk_recovery(Msg, R, S, Qx, Qy):
+    
+    sign = Signature(R,S)
+    pks = sign.recover_public_keys(digest_integer(Msg), curve_192, generator_192, generator_192.order())
+
+    print_("Test pk recover")
+
+    if len(pks) > 0:
```

Comment:
non empty arrays have a True value, so `if pks:` is equivalent and more pythonic in this case

Reply:
ok